src/index.ts

```diff
@@ -1 +1,2 @@
-export { SQSMessage, Consumer, ConsumerOptions } from './consumer';
+export { Consumer, ConsumerOptions } from './consumer';
+export { Message } from '@aws-sdk/client-sqs';
```

Comment:
Don't see any reason to export this, client can do this themselves if they need it. Didn't really need to export `SQSMessage` before it as well to note.